jmclient/jmclient/wallet_service.py

```diff
@@ -330,6 +331,7 @@ def sync_wallet(self, fast=True):
         # Don't attempt updates on transactions that existed
         # before startup
         self.old_txs = [x['txid'] for x in self.bci.list_transactions(100)]
+        self.bci.post_sync_wallet_callback(self.wallet)
```

Comment:
So this seems fine, just wondering whether the self.synced state might affect certain such callbacks; note we still use the paradigm in some scripts of doing wallet sync, err, synchronously and in a loop: "call sync_wallet repeatedly until it succeeds". So in some cases at least it's quite normal to get a failure here, the callback should know it might get called like that.

For this particular new callback method that's not a problem of course.

Reply:
Good point. I'll add a note to the doc string.